pkg/codegen/testing/test/testdata/config-variables-pp/go/config-variables.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		cfg := config.New(ctx, "")
+		requiredString := cfg.Require("requiredString")
+		requiredInt := cfg.RequireInt("requiredInt")
+		requiredFloat := cfg.RequireFloat64("requiredFloat")
+		requiredBool := cfg.RequireBool("requiredBool")
+		requiredAny := cfg.RequireObject("requiredAny")
```

Comment:
This is still broken, `RequireObject` should take another argument as input to mutate, which needs to be generated and handled in a special way